integration-tests/gradle/src/test/java/io/quarkus/gradle/LaunchUtils.java

```diff
@@ -21,4 +23,18 @@ protected static Process launch(Path jar, File output) throws IOException {
         return processBuilder.start();
     }
 
+    protected static void dumpFileContentOnFailure(final Callable<Void> operation, final File logFile,
```

Comment:
No need to have this `protected`, just make it `public` please

Reply:
Okk